### ISSUE TITLE
Error when pushing to Moz phabricator

### ISSUE THREAD

**Issue by djg** [mystor/phlay#72]
When pushing patches for review, `phlay` errors with:
```
error conduit[ERR-CONDUIT-CORE]: Unexpected exception when instantiating class.
```
when requesting review of the just published patches. Checking BZ and phabricator, patches haven't been updated.

Tail of console messages:
```
Publishing Patch Bug 1616171 - P1: Remote obsolete RemoteFrameChild fwd decl. r?nika
    Diff URI: https://phabricator.services.mozilla.com/differential/diff/233085/
Publishing Patch Bug 1616171 - P2: nsILoadInfo::targetBrowsingContext attribute. r?mattwoodrow,nika
    Diff URI: https://phabricator.services.mozilla.com/differential/diff/233086/
Publishing Patch Bug 1616171 - P3: Move initatorType from DocumentChannelCreationArgs to BC. r?mattwoodrow,nika
    Diff URI: https://phabricator.services.mozilla.com/differential/diff/233087/
Requesting Review Bug 1616171 - P1: Remote obsolete RemoteFrameChild fwd decl. r?nika
error conduit[ERR-CONDUIT-CORE]: Unexpected exception when instantiating class.
```

**Comment by AlaskanEmily** [comment]
I've been getting this error as well.

**Comment by aosmond** [comment]
The patch is uploaded but it can't create a new review from it. If you go to the diff URI and try to do it manually, it fails with the same error. I was able to use moz-phab to get the same patch put up for review. Phlay was unable to update the now existing review.

**Comment by julienw** [comment]
Same issue here as well.

**Comment by mystor** [comment]
This should be fixed with 5ebf34873b1b06c7037c1c62b441cd66bd82c74f, released in (Version 0.2.3)
